import_musicbrainz_row_ids.py

```diff
@@ -0,0 +1,41 @@
+from listenbrainz import config, db
+from brainzutils import musicbrainz_db
+
+import listenbrainz.db.user as db_user
+import sqlalchemy
+
+def import_musicbrainz_rows(musicbrainz_db_uri):
+    musicbrainz_db.init_db_engine(musicbrainz_db_uri)
+    db.init_db_connection(config.SQLALCHEMY_DATABASE_URI)
+    users = db_user.get_all_users()
+    with musicbrainz_db.engine.connect() as mb_connection:
+        for user in users:
+            name = user['musicbrainz_id']
```

Comment:
Just a suggestion, you can `continue` if `user['musicbrainz_row_id']` is already set, if you think it's possible the script may have to be run more than once (so it'll be much faster subsequent times).

Though if you make sure new accounts are created with a `musicbrainz_row_id` set, and deploy that patch first, this should only need to be run once.